infra/aws/terraform/management-account/iam-bentheelder.tf

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+
+resource "aws_iam_user" "bentheelder" {
+  name = "bentheelder"
+}
+resource "aws_iam_user_policy_attachment" "bentheelder_billing" {
+  user       = aws_iam_user.bentheelder.name
+  policy_arn = "arn:aws:iam::aws:policy/AWSBillingReadOnlyAccess"
+}
+resource "aws_iam_user_login_profile" "bentheelder_login" {
+  user    = aws_iam_user.bentheelder.name
+  password_reset_required = true
```

Comment:
I will provide you with an initial password @BenTheElder 
You will then be required to change it upon login.